### DISCUSSION TITLE
Integrate with Cursor editor

### DISCUSSION THREAD

**Discussion by vleytman** [spatie/ray#849] [Q&A] [answered]
Hi!

Is it possible to make files open in Cursor editor (https://cursor.so)? It's a fork of VSCode, so should probably be possible?

Thanks!

**Comment by conveydk** [comment, marked as answer]
Hi,
I've managed to setup Ray to use Cursor:
In Settings -> Editor -> Custom URL add this: cursor://file/%path:%line
You should be all set.

**Reply by JiveDig** [reply]
Worked for me, TY!

**Reply by jeffreyvr** [reply]
Same goes for Windsurf `windsurf://file/%path:%line` (for anyone who is searching for that, like me)
